client/state/data-layer/wpcom/sites/gutenberg/index.js

```diff
@@ -0,0 +1,40 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { EDITOR_TYPE_SET } from 'state/action-types';
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { registerHandlers } from 'state/data-layer/handler-registry';
+
+export const setType = action =>
+	http(
+		{
+			path: `/sites/${ action.siteId }/gutenberg`,
+			method: 'POST',
+			apiNamespace: 'wpcom/v2',
+			query: {
+				editor: action.editor,
+				http_envelope: 1,
```

Comment:
what's the envelop for?

Reply:
I thought that was our convention for ensuring all API requests return as 200 for the browser, for... reasons? Didn't realize it was optional, removing.